content/blog/data-trail/index.Rmd

```diff
@@ -0,0 +1,118 @@
+---
+output: hugodown::hugo_document
+
+slug: data-trail
+title: Solutions for R4DS, 2e with Data Trail
+date: 2023-08-02
+author: Jabir Ghaffar, Davon Person, Mine Çetinkaya-Rundel, Tracy Teal
+description: >
+    Jabir and Davon from Data Trail worked with us to create the solutions manual for R for Data Science, 2nd edition.
+    This blog post summarizes their experience and shares their reflections from working on this project.
+
+photo:
+  url: https://unsplash.com/photos/BnwRf_m3EIg
+  author: Tim Foster
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [learn] 
+tags: [book, internship]
+---
+
+```{=html}
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+```
+Last year at rstudio::conf(2022) Jeff Leek [shared about the Data Trail program](https://youtu.be/Vf301YCxP1Q).
+
+> DataTrail is a no-cost, paid 14-week educational initiative for young-adult, high school and GED-graduates. DataTrail aims to equip members of underserved communities with the necessary skills and support required to work in the booming field of data science. DataTrail is a fresh take on workforce development that focuses on training both Black, Indigenous, and other people of color (BIPOC) interested in the data science industry ***and*** their potential employers.
```

Comment:
What clause does "and their potential employers" apply to?

Reply:
I'm not sure, but this is a direct quote from https://www.datatrail.org/.